modules/mapper-extras/src/main/java/org/elasticsearch/index/mapper/RankFeatureFieldMapper.java

```diff
@@ -114,11 +114,11 @@ public Query existsQuery(QueryShardContext context) {
         }
 
         @Override
-        public ValueFetcher valueFetcher(MapperService mapperService, SearchLookup searchLookup, String format) {
+        public ValueFetcher valueFetcher(QueryShardContext context, SearchLookup searchLookup, String format) {
```

Comment:
If we're passing QueryShardContext I think that means we can avoid passing SearchLookup here?

Reply:
see my comment above, I don't think so, because we need to pass through the fetch lookup that is created in FetchContext?